doc-surrealdb_versioned_docs/version-latest/surrealql/statements/create.mdx

```diff
@@ -28,17 +28,11 @@ CREATE [ ONLY ] @targets
 
 ## Creating a Table Record
 
-When using the create statement the first word after `CREATE` is the table name. You also specify a record identifier using the `:` followed by a value. Ex. `CREATE Table:this`. The two together form the full [record ID](/docs/surrealdb/surrealql/datamodel/ids) which can be used to query the created data or by using the [`SELECT`](/docs/surrealdb/surrealql/statements/select) statement. See the [record ID](/docs/surrealdb/surrealql/datamodel/ids) page to learn more about what counts as a valid record identifier.  If no record identifier is specified, a random identifier will be generated.
-
-:::note
-__Note:__ You can also assign the recordID to be [randomly generated](/docs/surrealdb/surrealql/datamodel/ids#generating-record-ids) 
-:::
-
-The query below will create a new person table and also create a new record with a random ID.
+`CREATE` can be used with just a table name, in which case its ID will be generated randomly.
 
 ```surql
--- Create a new table
-CREATE person
+-- Create a new record
```

Comment:
@Dhghomon I wonder if we can add a comment here because the `CREATE person` will also implicitly create a schemaless table that can be seen in the `info for db`